content/wiki/formatting_scheme.md

```diff
@@ -21,6 +21,21 @@ Shaders are usually licensed under the former two licenses OR the Unlicense.
 Most Linebender projects files are written in Rust.
 For those files, the formatting produced by `rustfmt` is required.
 
+The following rustfmt config is recommended:
+
+```toml
+# LINEBENDER RUSTFMT CONFIG - v1
+# Ensure lines end with \n even if the git configuration core.autocrlf is not set to true
+newline_style = "Unix"
+
+# `Foobar { foo, bar }` is more readable than `Foo { foo: foo, bar: bar }`
+use_field_init_shorthand = true
+
+# Forces let else blocks to always be their own line(s)
+single_line_let_else_max_width = 0
```

Comment:
If we do decide to set this, I'd expect it to have the same value as `single_line_if_else_max_width`, I think.

Reply:
Indeed, they seem very similar and have identical default values.